### PR TITLE
Add Footer README

### LINKED ISSUE #913: Create Footer component Readme.md
**Is your feature request related to a problem? Please describe.**
We currently don't have explanations of what each component does, how it should be used, how it shouldn't be used.

We should add these explanations for the Footer component. 

This is a prerequisite to moving components to an external component library. 

**Describe the solution you'd like**
We should have a `README.md` file in the `src/app/components/Footer/` directory. 
It should have these sections.

```
- ## Description (high-level summary that will be accessible to product/UX/Test, plus link to Storybook, explain the different cases)
- ## When to use this component (including where it is currently used)
- ## When to not use this component (if this is not known, can leave the section empty)
- ## Accessibility notes (using accessiblity notes on Zeplin https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5babb3dea83a3937890e374c, ensuring they make sense to non-technical audiences)
- ## Roadmap (what we have planned for this component)
- ## Additional notes (link to any relevant ADRs)
```

**Describe alternatives you've considered**
We have considered other sections in the readme, but these seem like a good starting point. We can iterate/add/change sections in future, if needed. 

**Testing notes**
The `README.md` file should be clear, understandable, to people who are not developers and give a good explanation about the scope of a component. 

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by jtart**
Resolves #913 

_Adds a README for the footer component._

- ~~[ ] Tests added for new features~~
- ~~[ ] Test engineer approval~~

### CODE REVIEW

File: src/app/components/Footer/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Footer
+
+[Storybook link](https://simorghstorybook.now.sh/?selectedKind=Footer)
```
Comment (dr3) [inline src/app/components/Footer/README.md]: Could this be consistent with https://github.com/BBC-News/simorgh/pull/947/files#diff-b3af0b95ae347c7f3a6fc2e0dd68e2ecR1?

File: src/app/components/Footer/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Description
+
+The `Footer` is intended to be used at the bottom of an article. It is made up of a [`Brand banner`](https://github.com/BBC-News/simorgh/blob/footer-readme/src/app/components/Banner/index.jsx) and links to other BBC pages. These links are generally intended to be to more general and legal BBC pages, but do not have to be. It also includes information about copyright.
```
Comment (dr3) [inline src/app/components/Footer/README.md]:
```suggestion
The `Footer` is intended to be used at the bottom of an article. It is made up of a [`Brand banner`](../Brand) and links to other BBC pages. These links are generally intended to be to more general and legal BBC pages, but do not have to be. It also includes information about copyright. 
```

File: src/app/components/Footer/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Footer
```
Comment (bcmn) [inline src/app/components/Footer/README.md]:
```suggestion
# Footer [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind= Footer)
```

It'd be nice if this matched the precedence set by the [Header README](https://github.com/BBC-News/simorgh/blob/c8dad1f1c1bd5acc29f79588196c314cd164f8a1/src/app/components/Header/README.md).

File: src/app/components/Footer/README.md (hunk 1 above)
Comment (bcmn) [inline src/app/components/Footer/README.md]:
```suggestion
```

See above.

**Review by bcmn — COMMENTED** [review]
Looks great -- only a couple small comments.

File: src/app/components/Footer/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+## When to not use this component
+
+It should not be used when the brand is not `BBC News`.
+
+## Accessibility notes
+
+The component's parent element that it exports is a [`footer` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer), which typically contains information about the author and other connected content. The links are wrapped in a [`contentinfo` landmark](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/contentinfo.html), which denotes content around copyright, privacy and related content.
+
+## Roadmap
+- Removal of hard-coded `BBC News` brand - although this is not directly related to this component; will be done in the [Banner](https://github.com/BBC-News/simorgh/blob/footer-readme/src/app/components/Banner/index.jsx)
```
Comment (dr3) [inline src/app/components/Footer/README.md]:
```suggestion
- Removal of hard-coded `BBC News` brand - although this is not directly related to this component; will be done in the [Brand](../Brand)
```

File: src/app/components/Footer/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Footer [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind= Footer)
```
Comment (dr3) [inline src/app/components/Footer/README.md]:
```suggestion
# Footer [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Footer)
```

**Review by bcmn — DISMISSED** [review]

File: src/app/components/Footer/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Footer [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Footer)
+
```
Comment (dr3) [inline src/app/components/Footer/README.md:2]:
```suggestion
```
lol soz its tiny

**Review by dr3 — APPROVED** [review]

**Review by bcmn — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**